assets/scripts/embedding_tests_dev_i18n_0.0.1.html

```diff
@@ -1,10 +1,38 @@
 <h2>v0.0.1 (unminified)</h2>
```

Comment:
Comments from the other file apply to this one too.

Reply:
done.